templates/emails/new_user.html

```diff
@@ -0,0 +1,25 @@
+<p>Hi there!</p>
+
+<p>Congrats! You're now a Django Girls organizer, so awesome! Here is access to your
+website:</p>
+
+<p>Your website address is http://djangogirls.org/{{ event.eventpage.url }}</p>
+
+<p>
+  Login here:<br />
+  http://djangogirls.org/admin/
+</p>
+
+<p>
+  Email: {{ user.email }}<br />
+  Password: {{ password }}
+</p>
+
+<p>Here you can find instructions on editing website: http://organize.djangogirls.org/website/</p>
+
+<p>We also invited you to Django Girls Slack, where we chat, communicate and meet each other!</p>
+
```

Comment:
Is there a standard response template that I can copy? Or maybe you can update those templates here and push to this branch?